bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,568 @@
+<?xml version="1.0" ?>
+<thing:thing-descriptions bindingId="wundergroundupdatereceiver"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="last-received-datetime">
+		<item-type>DateTime</item-type>
+		<label>Last Received</label>
+		<description>The date and time of the last update.</description>
+		<category>Time</category>
+		<tags>
+			<tag>Point</tag>
+			<tag>Timestamp</tag>
+		</tags>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS"/>
+	</channel-type>
+
+	<channel-type id="datetime-utc" advanced="true">
+		<item-type>String</item-type>
+		<label>Last Updated</label>
+		<description>The date and time of the last update in UTC as submitted by the weather station. This can be 'now'.</description>
+		<category>Time</category>
+		<tags>
+			<tag>Point</tag>
+			<tag>Timestamp</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wind-speed">
+		<item-type>Number:Speed</item-type>
+		<label>Current Wind Speed</label>
+		<description>Current wind speed, using software specific time period.</description>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="wind-gust-speed">
+		<item-type>Number:Speed</item-type>
+		<label>Current Gust Speed</label>
+		<description>Current wind gust speed, using software specific time period.</description>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="wind-gust-direction">
+		<item-type>Number:Angle</item-type>
+		<label>Gust Direction</label>
+		<description>Current wind gust direction expressed as an angle using software specific time period.</description>
+		<category>Wind</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Wind</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="wind-speed-avg-2min" advanced="true">
+		<item-type>Number:Speed</item-type>
+		<label>Wind Speed 2min Average</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

If this is in a channel group, you could probably remove "Wind".

Reply:
There are also gust speed versions of this parameter, so wind is not superfluous.